### ISSUE TITLE
Wall-clock performance issues

### ISSUE THREAD

**Issue by verbiiz** [leaderj1001/LambdaNetworks#2]
LambdaResNet50 in comparison to a normal ResNet50 from my testing is 7x slower. I am training on ImageNet, and I am comparing your LambdaResNet50 to a torch.hub ResNet50.

Is this to be expected?

**Comment by leaderj1001** [comment]
Perhaps the torch.einsum computation time is too slow.
I will find another way to reduce the computation time.

Thanks

**Comment by verbiiz** [comment]
yeah, that was the conclusion i came to as well. it's significantly faster than torch's einops still.
